test/core/end2end/fuzzers/api_fuzzer.cc

```diff
@@ -244,9 +241,11 @@ grpc_ares_request* my_dns_lookup_ares(
   r->addr = gpr_strdup(addr);
   r->on_done = on_done;
   r->addresses = addresses;
-  grpc_timer_init(
-      &r->timer, grpc_core::Duration::Seconds(1) + grpc_core::Timestamp::Now(),
-      GRPC_CLOSURE_CREATE(finish_resolve, r, grpc_schedule_on_exec_ctx));
+  GetDefaultEventEngine()->RunAfter(grpc_core::Duration::Seconds(1), [r] {
```

Comment:
See below for my suggestion about avoiding allocations, I think that can apply here too.

Reply:
Done.